lib/utils/index.js

```diff
@@ -418,6 +419,7 @@ module.exports = {
       categories,
       skipDynamicArguments,
       skipDynamicArgumentsReport,
+      skipCoreHandlers,
```

Comment:
Hmm. In my opinion we should replace `context.getSourceCode().getScope()` with `wrapContextToOverrideTokenMethods()` rather than skip core's handler.
But I think we can merge this once to work around the current bug. I think we'll have to refactor after that.